accounts-db/src/append_vec.rs

```diff
@@ -928,7 +932,10 @@ impl AppendVec {
                         Self::get_type(bytes_subset, next).unwrap();
                     let (_hash, next): (&AccountHash, _) =
                         Self::get_type(bytes_subset, next).unwrap();
-                    let stored_size_aligned = u64_align!(next + (meta.data_len as usize));
+                    let stored_size_aligned = next
```

Comment:
Make the next element offset calculation to throw the matched error string. This makes test assert panic string easier.